src/main/kotlin/no/nav/omsorgspenger/oppgave/Oppgave.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.omsorgspenger.oppgave
+
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+import no.nav.omsorgspenger.extensions.DateUtils
+import no.nav.omsorgspenger.extensions.StringExt.trimJson
+import org.intellij.lang.annotations.Language
+
+// Koder:  https://kodeverk-web.nais.preprod.local/kodeverksoversikt
+
+data class Oppgave(
```

Comment:
hvorfor er tema, behandlingsTema og behandlingstype optionals - disse må vel alltid settes?
Heller lage to objekter om  det nå brukes til to formål